packages/api-contract/src/index.ts

```diff
@@ -8,4 +9,5 @@ const c = initContract();
 export const contract = c.router({
   post: postContract,
   user: userContract,
+  organizer: organizerContract,
```

Comment:
Should we export the attendee contract mai?
```suggestion
  organizer: organizerContract,
  attendee: attendeeContract
```

Reply:
the attendee part isn't implemented yet since I'll pass it to Non, but we can add it for now.

Reply:
ok, change of plan. If you export it, the router will require the controller to be implemented, so we should ignore this part for now until I pass this to Non.